content/id/docs/concepts/storage/volume-snapshot-classes.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Volume Snapshot Class
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Laman ini menjelaskan tentang konsep VolumeSnapshotClass pada Kubernetes. Sebelum melanjutkan,
```

Comment:
VolumeSnapshotClass -> `VolumeSnapshotClass`

Reply:
the original page styling is incorrect, for resource name it should be camel case according to https://kubernetes.io/docs/contribute/style/style-guide/#use-camel-case-for-api-objects